### DISCUSSION TITLE
Next js shows blank on React native webview android

### DISCUSSION THREAD

**Discussion by JogiPrasadPakki** [vercel/next.js#18700] [Help]
Hello i want to establish communication between react-native webview and nextjs using window.onMessage()   but after start **yarn dev**  webivew shows blank page and tested on android chrome browser and it working and result from yarn build  also work but i want debug with **yarn dev** 
My Android Version : 4.4 Kitkat

**Comment by getsettalk** [comment]
hey dis you find any soution
